docs/source/model_design/risk_factors.rst

```diff
@@ -134,8 +134,131 @@ Definitions
 Theoretical Minimum Risk Exposure Level/Distribution (TMREL/D)
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+The **theoretical minimum risk exposure level (TMREL)** is the level of risk
+exposure that would minimize the risk of an adverse outcome for an individual.
+For example, the TMREL for smoking would be "has never smoked." The
+corresponding concept on the population level is the **theoretical minimum risk
+exposure distribution (TMRED)**, which is the distribution of risk exposure that
+would yield the lowest possible population risk. For smoking, the TMRED would be
+the trivial probability distribution assigning everyone in the population to the
+TMREL category "has never smoked." [WHO-Global-Health-Risks-Annex]_,
+[GBD-2017-Risk-Appendix]_
+
+Recall from the `causality section <What is a causal relationship?_>`_ that
+counterfactual analysis is used to describe the causal relationship between a
+risk factor and an outcome. **The TMRED is a particular choice of counterfactual
+exposure distribution** used for the causal attribution of disease burden to a
+given risk factor in a population (see `Population Attributable Fraction
+(PAF)`_). Other choices of counterfactual include the *plausible* minimum risk,
+*feasible* minimum risk, and *cost-effective* minimum risk, each of which can
+obviously depend on specific attributes of the population under consideration.
+On the other hand, Murray et al. state
+[Comparative-quantification-health-risks]_:
+
+  Biological principles as well as considerations of equity would necessitate
+  that, **although the exposure distribution for theoretical minimum risk may
+  depend on age and sex, it should in general be independent of geographical
+  region or population.**
+
+However, they go on to add:
+
+  Exceptions to this are however unavoidable. An example would be the case of
```

Comment:
I'm a little confused by this.

Would the exceptions be due to biological differences that are region-specific, or feasible differences that are geographically specific?

For example, maybe going for a 30 min jog outside every day is good for your heart and respiratory health, unless there are wildfires and SMOG, in which case it ends up being bad for you?

So here the TMRED would be geographically dependent, but any given population, if transplanted to the same geography, would presumably have the same TMRED

Which is different from, say, if there was some specific population with a biological immunity to skin cancer, which then maybe has a different TMRED for sun exposure than the rest of the world?